### PR TITLE
concepticon link for v0.2

### PR DESCRIPTION
**Pull request by Tarotis**
hello @LinguList,

I updated the both raw- and metadata in order for the integration of the freshly added concepticon concept-list of the original source. I was a bit insecure about the relevant lines in the python-file, as it was necessary to use `concept.attributes["spanish"]` instead of `concept.spanish`, but as the workflow run smoothly, I guess thats more a question of aesthetics than of actual relevance. Still, I would really appreciate if you could take a look at the changes.

Sidenote: The concepticon-version used for this version was the branch that was used for adding the concept list, as concepticon 2.5 is not released yet. I will change this once concepticon 2.5 is released.

### CODE REVIEW

**Comment by Tarotis** [comment]
The conflicts probably arise due to the fact that I commited to the main branch instead of creating a new working branch for adding v0.2. I then realized that I should probably do that and want to undo the last commit to /main/, but I don't quite understand how to do so.

File: lexibank_constenlachibchan.py [hunk 1]
```diff
@@ -12,6 +12,11 @@
 class CustomLexeme(Lexeme):
     CU = attr.ib(default=None)
 
+@attr.s
+class CustomConcept(Concept):
+    Page_in_Source = attr.ib(default=None)
+    Number = attr.ib(default=None)
+    English_Gloss = attr.ib(default=None)
```
Comment (LinguList) [inline lexibank_constenlachibchan.py:19]: English gloss is typically the Name attribute, so no need to add it here.

File: lexibank_constenlachibchan.py [hunk 2]
```diff
@@ -26,33 +31,39 @@ class Dataset(BaseDataset):
...
+
+        concept_lookup = {}
+        for concept in self.conceptlists[0].concepts.values():
+            idx = concept.id.split("-")[-1] + "_" + slug(concept.attributes["spanish"])
             args.writer.add_concept(
                     ID=idx,
-                    Name=concept['ENGLISH'],
-                    Concepticon_ID=concept['CONCEPTICON_ID'],
-                    Concepticon_Gloss=concept['CONCEPTICON_GLOSS']
+                    Name=concept.attributes["spanish"],
```
Comment (LinguList) [inline lexibank_constenlachibchan.py:44]: concept.gloss is better here.

File: lexibank_constenlachibchan.py [hunk 3]
```diff
@@ -26,33 +31,39 @@ class Dataset(BaseDataset):
...
+                    English_Gloss=concept.gloss,
```
Comment (LinguList) [inline lexibank_constenlachibchan.py:45]: as I said before, English_Gloss is not what we typically use, so leave it out.

**Review by LinguList — APPROVED** [review]
so basically fine, apart from my small comments on English_gloss, etc.

File: lexibank_constenlachibchan.py (hunk 2 above)
Comment (Tarotis) [reply lexibank_constenlachibchan.py:44]: Would you suggest adding `SPANISH` as an additional column instead?

File: lexibank_constenlachibchan.py (hunk 3 above)
Comment (Tarotis) [reply lexibank_constenlachibchan.py:45]: Would you suggest adding `SPANISH` as an additional column instead?

**Comment by LinguList** [comment]
We use Spanish_Gloss, or Chinese_Gloss in these cases, so we preserve 
the info from there. So yes!

**Pull request merged**